packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/TaskNavButtonsContainer.spec.js

```diff
@@ -4,28 +4,36 @@ import { observable } from 'mobx'
 import { expect } from 'chai'
 import sinon from 'sinon'
 import TaskNavButtonsContainer from './TaskNavButtonsContainer'
+import ClassificationStore from '../../../../../../../../store/ClassificationStore'
```

Comment:
Are we still waiting to merge the PR that aliases these to `@store/ClassificationStore` etc?